.github/workflows/release.yml

```diff
@@ -0,0 +1,117 @@
+name: Publish to crates.io
+
+# on:
+#   push:
+#     tags:
+#       - 'v*'
+
+on:
+  pull_request: # Test trigger
+
+env:
+  RUSTFLAGS: -Dwarnings
+  RUST_BACKTRACE: 1
+  # Change to specific Rust release to pin
+  rust_stable: stable
+  rust_nightly: nightly-2024-07-07
+  # When updating this, also update relevant docs
+  rust_min: '1.81'
+  RELEASE_ROLE_ARN: ${{ secrets.RELEASE_ROLE_ARN }}
+  TOKEN_KEY_NAME: ${{ secrets.TOKEN_KEY_NAME }}
+  AWS_REGION: us-west-2
+  # Test values
+  TEST_DRY_RUN: 'true'
```

Comment:
simply `DRY_RUN` would work but also probably should be an input for workflow dispatch

Reply:
yeah, and probably give a default input value to `true` so that it lets the user to opt-in for production release.